velox/substrait/tests/VeloxSubstraitRoundTripTest.cpp

```diff
@@ -313,6 +312,29 @@ TEST_F(VeloxSubstraitRoundTripTest, notNullLiteral) {
       plan, "SELECT true, 23, 45, 678, 910, 1.23, 4.56, '789'");
 }
 
+TEST_F(VeloxSubstraitRoundTripTest, arrayLiteral) {
+  auto vectors = makeRowVector(ROW({}, {}), 1);
+  auto plan = PlanBuilder(pool_.get())
+                  .values({vectors})
+                  .project({"array[0, 1, 2, 3, 4]"})
+                  .planNode();
+  assertPlanConversion(plan, "SELECT array[0, 1, 2, 3, 4]");
```

Comment:
Would you also test empty arrays and arrays with a mix of null and non-null values? SELECT array[], array[1, 2, null].